cluster/libvirt-coreos/util.sh

```diff
@@ -22,7 +22,8 @@ source "$ROOT/${KUBE_CONFIG_FILE:-"config-default.sh"}"
 source "$KUBE_ROOT/cluster/common.sh"
 
 export LIBVIRT_DEFAULT_URI=qemu:///system
-
+export SERVICE_ACCOUNT_LOOKUP=${SERVICE_ACCOUNT_LOOKUP:-false}
+export ADMISSION_CONTROL=${ADMISSION_CONTROL:-NamespaceLifecycle,NamespaceAutoProvision,LimitRanger,ServiceAccount,ResourceQuota}
```

Comment:
I think NamespaceAutoProvision is deprecated (@derekwaynecarr, confirm?), and should probably be removed from local-up-cluster.sh as well.
@pmorie I see SecurityContextDeny in some config-default.sh scripts and not others... is it expected to be included by default or not?